implementations/java-jsonschema-validator/build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'java'
+}
+
+sourceSets {
+    main {
+        java {
+            srcDir '.'
+            include 'BowtieJsonSchemaValidator.java'
+        }
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+
+def harnessImplementation = [
+    group: 'com.networknt',
+    name: 'json-schema-validator',
+    version: '1.0.85'
+]
+
+def jsonProvider = [
+        group: 'com.fasterxml.jackson.core',
+        name: 'jackson-databind',
+        version: '2.15.2'
+]
+
+dependencies {
+    implementation "$harnessImplementation.group:$harnessImplementation.name:$harnessImplementation.version"
+    implementation "$jsonProvider.group:$jsonProvider.name:$jsonProvider.version"
```

Comment:
This whole "json provider" dependency probably should be removed, because the "networknt" by itself depends on other json libraries (I mean, it just happens that it also uses `jackson` libraries) and all the transitive dependencies will be handled automatically. 
So, the line 
```groovy
implementation "$harnessImplementation.group:$harnessImplementation.name:$harnessImplementation.version"
```
will be enough